buildscripts/ci/macos/setup.sh

```diff
@@ -30,6 +30,7 @@ echo "Install build tools"
 rm '/usr/local/bin/2to3' # fixing install python 3.9 error (it is a dependency for ninja)
 brew install ninja pkg-config --quiet
 brew install cmake --formula --quiet
+arch --x86_64 /bin/bash -c "brew install jack --formula --quiet"
```

Comment:
This is not going to work. You need a "universal" binary of Jack, ie x86_64 and arm64 combined, and HomeBrew is not going to provide that. An additional requirement is that the deployment target needs to be set to macOS 10.15 at most. 

When we are going to merge this, I can help with obtaining a suitable binary, but it's not entirely trivial.

Reply:
Thanks! Perhaps try install jack from source?

Reply:
It looks like that should work, but you have to un-tar the downloaded archive.

Reply:
oh! thanks! missed that step